src/Management/src/Prometheus/PrometheusExtensions.cs

```diff
@@ -25,11 +30,39 @@ public static class PrometheusExtensions
     /// The incoming <paramref name="services" /> so that additional calls can be chained.
     /// </returns>
     public static IServiceCollection AddPrometheusActuator(this IServiceCollection services)
+    {
+        return AddPrometheusActuator(services, true, null);
+    }
+
+    /// <summary>
+    /// Adds the services used by the Steeltoe-configured OpenTelemetry Prometheus exporter.
+    /// </summary>
+    /// <param name="services">
+    /// The <see cref="IServiceCollection" /> to add services to.
+    /// </param>
+    /// <param name="configureMiddleware">
+    /// When <c>false</c>, skips configuration of the ASP.NET middleware pipeline. While this provides full control over the pipeline order, it requires
+    /// manual addition of the appropriate middleware for the Prometheus exporter to work correctly.
+    /// </param>
+    /// <param name="configureBranchedPipeline">
+    /// Optional callback to configure the branched pipeline. Called before registration of the Prometheus middleware.
+    /// </param>
+    /// <returns>
+    /// The incoming <paramref name="services" /> so that additional calls can be chained.
+    /// </returns>
+    public static IServiceCollection AddPrometheusActuator(this IServiceCollection services, bool configureMiddleware,
+        Action<IApplicationBuilder>? configureBranchedPipeline)
     {
         ArgumentNullException.ThrowIfNull(services);
```

Comment:
Should throw `InvalidOperationException` when `configureMiddleware` is `false` and `configureBranchedPipeline` is not null (and have a test for it).